src/Pages/Login.cshtml.cs

```diff
@@ -27,9 +29,17 @@ public class LoginModel : PageModel
         public bool FailedToSendEmail;
 
         private UserDatabase UserDatabase { get; }
-        public LoginModel(UserDatabase userDatabase)
+        private EmailSender EmailSender { get; }
+        public LoginModel(UserDatabase userDatabase, EmailSender emailSender)
         {
             UserDatabase = userDatabase;
+            EmailSender = emailSender;
+            UserDatabase.Registered += OnRegister;
+        }
+
+        private async void OnRegister(object sender, UserModel user)
+        {
+            await EmailSender.SendGreetingEmailAsync(user.Email);
```

Comment:
Why is greetings email in login? :D

Reply:
I have no idea :D Changed it to be On Login, even though it doesn't do anything, just to show that we could do something :D